packages/sdk/src/commons.ts

```diff
@@ -38,8 +38,9 @@ export const CHAIN_ADDRESSES: Record<ChainId, ChainAddresses> = {
 
 export const SUBGRAPH_URL: Record<ChainId, string | null> = {
     [ChainId.GNOSIS]:
-        "https://api.thegraph.com/subgraphs/name/carrot-kpi/carrot-kpi-gnosis",
-    [ChainId.SEPOLIA]: null,
+        "https://api.thegraph.com/subgraphs/name/carrot-kpi/carrot-gnosis",
+    [ChainId.SEPOLIA]:
+        "https://api.thegraph.com/subgraphs/name/carrot-kpi/carrot-sepolia",
     [ChainId.SCROLL_SEPOLIA]: null,
```

Comment:
If Scroll Sepolia has a subgraph aren't we missing the url here?

Reply:
It doesn't have a Carrot subgraph deployment. TheGraph supports Scroll Sepolia but I thought it would make sense to have at least one network without a subgraph for testing purposes (for example something that we're currently missing is the removal of the decentralization mode toggle for networks that don't have a Carrot subgraph deployment)